inc/command/class-command.php

```diff
@@ -66,10 +67,97 @@ protected function execute( InputInterface $input, OutputInterface $output ) {
 	protected function phpunit( InputInterface $input, OutputInterface $output ) {
 		$options = [];
 
+		// Get dev-tools config.
+		$config = $this->get_config()['phpunit'] ?? [];
+
+		// Set default directories.
+		$directories = [ 'tests' ];
+
+		// Get directories from config.
+		if ( isset( $config['directories'] ) ) {
+			$directories = array_merge( (array) $config['directories'], $directories );
+		}
+
+		$directories = array_map( function ( $path ) {
+			return trim( $path, DIRECTORY_SEPARATOR );
+		}, $directories );
+		$directories = array_filter( $directories, function ( $path ) {
+			$full_path = $this->get_root_dir() . DIRECTORY_SEPARATOR . $path;
+			return is_dir( $full_path );
+		} );
+		$directories = array_unique( $directories );
+
+		// Write XML config.
+		$doc = new DOMDocument( '1.0', 'utf-8' );
+
+		// Create PHPUnit Element.
+		$phpunit = $doc->createElement( 'phpunit' );
+		$phpunit->setAttribute( 'bootstrap', 'altis/dev-tools/inc/phpunit/bootstrap.php' );
```

Comment:
Should this have `get_root_dir()` applied too?

Reply:
No, this is parsed on the VM so we need to use relative paths. `get_root_dir()` gives you the host path.

Reply:
Gotcha. I thought it'd at least need `vendor/`?

Reply:
I'm outputting the file to the `vendor` directory as it's generated and shouldn't end up in VCS, the paths in the file are relative to that.